migrations/V83__Add_Accessibility_Notes.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE accessibility_notes (
```

Comment:
Would `accessibility_request_notes` be a better name? That would follow the pattern that we have for documents and status records.